sql_generators/glean_usage/templates/baseline_clients_last_seen.view.sql

```diff
@@ -6,7 +6,35 @@ AS
 SELECT
   {% for ut in usage_types %}
     `moz-fx-data-shared-prod`.udf.pos_of_trailing_set_bit(days_{{ ut }}_bits) AS days_since_{{ ut }},
-  {% endfor %}
+  {% endfor %},
+  -- Metrics based on activity
+  mozfun.bits28.days_since_seen(days_active_bits) = 0 AS is_dau,
+  mozfun.bits28.days_since_seen(days_active_bits) < 7 AS is_wau,
+  mozfun.bits28.days_since_seen(days_active_bits) < 28 AS is_mau,
+  -- Metrics based on pings sent
+  mozfun.bits28.days_since_seen(days_seen_bits) = 0 AS is_daily_user,
+  mozfun.bits28.days_since_seen(days_seen_bits) < 7 AS is_weekly_user,
+  mozfun.bits28.days_since_seen(days_seen_bits) < 28 AS is_monthly_user,
+  CASE
+    WHEN LOWER(isp) = 'browserstack'
+      THEN CONCAT(app_name, ' ', isp_name)
+    {% if app_name = "fenix" %}
+    WHEN LOWER(distribution_id) = 'mozillaonline'
+      THEN CONCAT(app_name, ' ', distribution_id)
+    {% endif %}
+    ELSE app_name
+  END AS analysis_app_name,
```

Comment:
I think it's safe to leave this as `app_name`. That's consistent with how it's reported in `active_users_aggregates`:

```sql
SELECT DISTINCT app_name
  FROM `moz-fx-data-shared-prod.firefox_ios.active_users_aggregates`
 WHERE submission_date = "2024-04-04"
```
returns `Firefox iOS, Firefox iOS BrowserStack`

Reply:
~We have to keep in mind that this specific field in this table might be used elsewhere so we should be at the very least careful about swapping out this value.~

Actually, apologies. I just checked and app_name does not exist yet so it's fine to just call it app_name. I can see I will also need to include string values for it.